_layouts/home.html

```diff
@@ -6,7 +6,7 @@
     <section class="search-section">
         <div class="container">
             <h1 class="search-section__title">Welcome to the Spryker <br>documentation center</h1>
-            <div class="search-section__panel">
+            <div class="search-section__panel dropdown-menu--dark">
```

Comment:
According to BEM you can't use modifier class without base class. So please add `dropdown-menu` class.